### DISCUSSION TITLE
Media upload stopped working after library delete

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9329] [Q&A]
### The bug

Media upload stopped working after library delete

### The OS that Immich Server is running on

Windows 10 with wsl2

### Version of Immich Server

Latest

### Version of Immich Mobile App

Latest

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1. Using immich app on android, upload whats app video
2. Let the various jobs complete their tasks
3. Videos are available on app as well as laptop browser
4. I mistakenly deleted the library folder on immich server
5. New videos received and are uploaded on server
6. Manually run all the jobs from admin settings

Issue : Whats app videos available only in immich app. Junk thumbnails present on server. I was expecting that the thumbnails from server will disappear and immich app will upload (auto or manual) all the whats app videos once more.
```

**Comment by danieldietzler** [comment]
It won't be reuploaded because the asset still exists in the database, including a hash that will be used to detect duplicates. Thus, the app thinks those assets already exist on the server

**Reply by amgschnappi** [reply]
Ok. then how to force rescan of all data? So that new data remains on server and older/stale infos are reset?

**Reply by danieldietzler** [reply]
You can't "force rescan". You need to purge the data following the guide linked in the comment below by bo0tzz.

**Comment by bo0tzz** [comment]
Relevant: https://immich.app/docs/FAQ#how-can-i-purge-data-from-immich
